src/main/java/eu/bbmri/eric/csit/service/negotiator/database/repository/PostRepository.java

```diff
@@ -0,0 +1,11 @@
+package eu.bbmri.eric.csit.service.negotiator.database.repository;
+
+import eu.bbmri.eric.csit.service.negotiator.database.model.Post;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface PostRepository extends JpaRepository<Post, Long> {
+
+  Optional<Post> findByPosterId(String id);
```

Comment:
Person ID is of type Long

Reply:
Fixed in f963e8c6d1234c0e9622d3e889c9f0ead52b3cc2